app/assets/stylesheets/variables_and_mixins.scss

```diff
@@ -0,0 +1,40 @@
+//colors
+$white: #fff;
+$black_5: rgba(0, 0, 0, .05);
+$color_catskill_white: #f1f4f7;
+$color_storm_dust: #5f6468;
+$color_tapa: #777;
+$color_dodger_blue: #30a5ff;
+$color_gull_gray: #9fadbb;
+$color_java: #1ebfae;
+$color_yellow_orange: #ffb53e;
+$color_alizarin_crimson: #f9243f;
+$color_coral_red: #ef4040;
+$color_gallery: #eee;
+$black_10: rgba(0, 0, 0, .1);
+$color_athens_gray: #e9ecf2;
+$color_elephant: #1b3548;
+$color_atlantis: #8ad919;
+$white_75: rgba(255, 255, 255, .75);
+$color_mountain_mist: #999;
+$white_80: rgba(255, 255, 255, .8);
+$white_20: rgba(255, 255, 255, .2);
+$white_50: rgba(255, 255, 255, .5);
+$color_tundora: #444;
+$color_fuscous_gray: #555;
+$color_alabaster: #f9f9f9;
+$black_50: rgba(0, 0, 0, .5);
+$color_st_tropaz: #285e8e;
+$black_25: rgba(0, 0, 0, 0.25);
+$color_black_haze: #f6f6f6;
+$color_alto: #ddd;
+$color_mercury: #e6e7e8;
+
+
+//Methods
+
+@mixin border-radius($radius) {
```

Comment:
remove this 
and use http://compass-style.org/reference/compass/css3/border_radius/
where you need